packages/grammar/src/Happy/Grammar.lhs

```diff
@@ -123,4 +125,24 @@ In hindsight, this was probably a bad idea.
 > firstStartTok   = 3
 > dummyTok        = 2
 > errorTok        = 1
-> epsilonTok      = 0
\ No newline at end of file
+> epsilonTok      = 0
+
+-----------------------------------------------------------------------------
+Replace $$ with an arbitrary string, being careful to avoid ".." and '.'.
+
+> mapDollarDollar :: String -> Maybe (String -> String)
+> mapDollarDollar code0 = go code0 ""
```

Comment:
How come this is moved here? It seems only used by backend code? I think we should leave in genutils at least temporarily, unless I am missing something.